TEAM_STRUCTURES.md

```diff
@@ -0,0 +1,228 @@
+  - **Development and Maintenance:**
+    - [JavaScript Core Dev](#javascript-core-dev)
+    - [Golang Core Dev](#golang-core-dev)
+    - [IPFS GUI](#ipfs-gui)
+    - [IPFS Infrastructure](#ipfs-infrastructure)
+    - [Dev Team Enablement (QA, Testing, Support)](#qa-testing-and-dev-team-enablement)
+  - **Community, Education and Usage:**
+    - [Community, Events & Evangelism](#community-events-evangelism)
+    - [Documentation, Education and Websites](#documentation-education-and-websites)
+  - **Ongoing Efforts to Support Specific Uses:**
+    - [Integration with Web Browsers](#integration-with-web-browsers)
+    - [Dynamic Data and Capabilities](#dynamic-data-and-capabilities)
+    - [IPFS Cluster](#ipfs-cluster)
+- [Existing Research Groups](#existing-Research-groups)
+
+## Description
+
+The IPFS Project is developed by multiple teams formed by multiple organizations and ponctual or continuous invidividual contributors. A structure was developed (and continues to evolve) around Working & Research Groups, OKRs and PM best practices that were designed to support the already large and growing group of people, to coordinate among eachother at a global scale.
+
+This document captures the Team Structures that emerged and that led to the many of the sucesses within the IPFS project, namely: Working Groups & Research Groups. At the end of this document, you will find pointers to Teams and Projects that have growned to become their own entities, namely: IPLD, libp2p and Multiformats.
+
+#### One line definition
+
+- **Working Group** - A cohesive team with measurable goals for each quarter. A Working Group needs to have a Captain, a set of people commited to it, and entry point repo and Objectives & Key Results defined per Quarter.
+- **Research Group** - A set of people focused on exploring a specific vertical and output possible research surveys, tools, experiments and other types of knowledge that can inform and help the work done by the Working Groups or spark the creation of new Working Groups all together.
+
+#### Expanded description
+
+**Working Groups**
+
+Working Groups are teams of people that are appointed to research, develop and deploy work under the scope that the working group is focused. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
+
+Each Working Group is free to experiment with setting their own pace, tracking work and defining priorities. The only requirements are that the Working Group exposes its focus through OKRs to the rest of the org (common interface), that it assigns a Captain, creates an entry point repo and has at least 2 full time contributors dedicatted to it.
+
+Each contributor shouldn't carry responsibilities accross multiple working groups. This is not forbidden by any means but it is greatly disencouraged as it will disable the contributor from achieving full focus.
```

Comment:
This is a bit confusing as what seems to be outlined as Working Groups here, is what I believed "Teams" was before. 

One of the OKRs for the Testing WG this quarter is "Have at least one member from each team be a part of the Working Group" which is based on the fact that I thought Working Groups are supposed to have members who can also be a part of other working groups. I think that's the "common" definition of a working group as well, as set of people who can participate in many working groups.

It's also unclear where the "IPFS Core" Working Group would fit into this here, since people activate in that, needs to be active in other working groups as well.

Reply:
> This is a bit confusing as what seems to be outlined as Working Groups here, is what I believed "Teams" was before.

There hasn't been a _Teams_ definition before.

> One of the OKRs for the Testing WG this quarter is "Have at least one member from each team be a part of the Working Group"

In reality, it is still OKRs of the Dev Team Working Group and not complete different contexts